### PR TITLE
Update generic-bigtreetech-skr-mini-e3-v3.0.cfg

### PR DESCRIPTION
**Pull request by ExplicitChimp**
Added Pin for Fillament runout sensor:  Pin PC15 on drawing title "BTT SKR MINI E3_MCU_V3.0" Version V3.0 Revision Date 13/10/2021.

### CODE REVIEW

**Comment by JamesH1978** [comment]
Thank you for submitting a PR, pleas refer to point 3 in "What to expect in a review" in https://github.com/Klipper3d/klipper/blob/master/docs/CONTRIBUTING.md and provide a signed off by line.

Thanks
James

File: config/generic-bigtreetech-skr-mini-e3-v3.0.cfg [hunk 1]
```diff
@@ -107,6 +107,9 @@ pin: PB15
 [fan]
 pin: PC6
 
+[filament_switch_sensor e0_sensor]
```
Comment (JamesH1978) [inline config/generic-bigtreetech-skr-mini-e3-v3.0.cfg:110]: Please comment these new lines, as a new user may not have a filament sensor, but allows them to have it as an option if needed.

Thanks
James

**Review by JamesH1978 — CHANGES_REQUESTED** [review]
Please see my comments in the code.

Thanks
James

File: config/generic-bigtreetech-skr-mini-e3-v3.0.cfg (hunk 1 above)
Comment (ExplicitChimp) [reply config/generic-bigtreetech-skr-mini-e3-v3.0.cfg]: Hi James,

I have done as requested, but in addition I have created a commented out section for a bed probe.  Both of these I have tested on this board.

Kind regards,

ExplicitChimp

File: config/generic-bigtreetech-skr-mini-e3-v3.0.cfg [hunk 2]
```diff
@@ -107,8 +107,26 @@ pin: PB15
...
 pin: PC6
 
-[filament_switch_sensor e0_sensor]
-switch_pin: PC15
+# If using a Filament Runout Sensor, uncomment the two lines below
+#[filament_switch_sensor e0_sensor]
+#switch_pin: PC15
+
+# If using bltouch or similar, uncomment and edit the lines below
+#[bltouch]
```
Comment (JamesH1978) [inline config/generic-bigtreetech-skr-mini-e3-v3.0.cfg:115]: Please only add basic options for bltouch, some of these are only to be used with certain probes, like clones. This is a board config not a printer config, please only state the pins and the offsets, it is up to the end user to properly set it for their touch probe.

Thanks
James

Comment (ExplicitChimp) [reply config/generic-bigtreetech-skr-mini-e3-v3.0.cfg:115]: Hi James, 

All done.

Kind regards,

ExplicitChimp

**Review by ExplicitChimp — COMMENTED** [review]
Had a whitespace error on line 115 now corrected.

**Review by JamesH1978 — APPROVED** [review]
Looks fine to me now,

thanks 
James

**Comment by KevinOConnor** [comment]
Thanks.  Looks fine to me, but I don't see a signed-off-by line.  https://github.com/Klipper3d/klipper/blob/master/docs/CONTRIBUTING.md

-Kevin

**Pull request closed without merging**